cms/categories/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from cms.categories import views
+
+urlpatterns = [
+    path("", views.CategoryListView.as_view()),
```

Comment:
This doesn't seem to work: I get redirected from categories to categories/ which matches against the line below with NoReverseMatch at /categories/
Reverse for 'category-detail' with arguments '('',)' not found. 1 pattern(s) tried: ['categories/(?P<slug>[-a-zA-Z0-9_]+)$']